desktop/index.md

```diff
@@ -78,15 +78,15 @@ using a secure repository.
         <div class="component">
              <div class="component-icon">
                  <img src="/assets/images/download.svg" alt="Download and install" width="70" height="70">
```

Comment:
Looks like the install icon is missing the link

```suggestion
                 <a href="/desktop/install/mac-install"><img src="/assets/images/download.svg" alt="Download and install" width="70" height="70"></a>
```

Reply:
Yeh I missed this one out on purpose the descriptions below provide the link to each install page for Mac, Windows, Linux and I wasn't sure which one to attach to the icon

Reply:
maybe we should remove the link for the h2 element too in that case? it changes the text color but I don't think it looks bad.

<img width="265" alt="image" src="https://user-images.githubusercontent.com/35727626/196680352-d0e22c41-9a71-4cc5-ab16-da25b1b6a55c.png">
<img width="222" alt="image" src="https://user-images.githubusercontent.com/35727626/196680393-844e4b22-9507-45cf-afbe-4b2c4714a098.png">

Reply:
I'm tempted to leave as is so users can access whichever is appropriate to them

Reply:
Fixing now!